go.mod

```diff
@@ -3,9 +3,12 @@ module github.com/containerd/nri
 go 1.14
 
 require (
+	github.com/containerd/cgroups v1.0.1
```

Comment:
thinking about whether the example content should affect the go.mod for the project

Reply:
but I do like that it's built tested..

Reply:
Updated. PTAL. Thanks!